lava-v2-jobs-from-api.py

```diff
@@ -96,10 +124,9 @@ def main(args):
                         exit(1)
             if build['kernel_image']:
                 # handle devices without a DTB, hacky :/
+                build['dtb_dir_data'].append('qemu')
```

Comment:
We have a couple of qemu devices on a separate dispatcher in the Collabora lab, so it's fine for us to run some jobs.  There aren't that many to run in fact with the limited defconfig set, and they're quick.  Having a cloud lab would still help I guess, to provide some redundancy.

I suppose you could still have qemu devices in you LAVA lab but not make them available to kernelci.  Or maybe this is another case of per-lab filtering (trees and device types).